### ISSUE TITLE
[Bugs?] await keyword mistake order when I use Swift-testing library.

### ISSUE THREAD

**Issue by psbss** [nicklockwood/SwiftFormat#1707]
Hello

Apple's new testing library [swift-testing]() may include from Xcode16.
So We started implementing of this library for our product but the format of `await` keyword order was mistake when I run swift-format command.

Examples

```swift
// original
#expect (await monitor.isAvailable == false)

// formatted (mistake)
#expect await (monitor.isAvailable == false)
```

Version: `v0.53.8` installed
This is our config:

```.swiftformat
# disable
--disable wrapMultilineStatementBraces
--disable preferKeyPath
--disable preferForLoop

# format options
--stripunusedargs closure-only
--self init-only
--commas inline
--extensionacl on-declarations
--wrapparameters before-first
--wraparguments before-first
--maxwidth 120
--guardelse next-line

# version
--swiftversion 5.9

```

Do we have a workaround?

**Comment by nicklockwood** [comment]
I suggest you use --disable hoistAwait for now

**Comment by psbss** [comment]
I see, thank you

**Comment by nicklockwood** [comment]
@psbss fixed in 0.54.0

**Comment by psbss** [comment]
Thank you so much @nicklockwood !!
